src/main/clojure/aerospike_clj/utils.clj

```diff
@@ -39,8 +40,18 @@
 
 (defn v->array
   "An optimized way to convert vectors into Java arrays of type `clazz`."
```

Comment:
I don't think this docstring is accurate. Sets and lists are also `Collection`s.

Reply:
Updated